src/composables/draw/drawn-features.composable.ts

```diff
@@ -0,0 +1,29 @@
+import useMap from '../map/map.composable'
+import { Collection, Feature } from 'ol'
+import { Geometry } from 'ol/geom'
+import VectorLayer from 'ol/layer/Vector'
+import VectorSource from 'ol/source/Vector'
+
+export default function useDrawnFeatures() {
+  const map = useMap().getOlMap()
+
+  const features = new Collection<Feature<Geometry>>()
+  const drawLayer = new VectorLayer({
```

Comment:
I would have put this layer creation code into the ol-layer-factory (and a new helper) so that we have all layer creation inside a uniq folder` services/ol-layer`, what do you think?

Reply:
👍🏽